MaxText/layers/quantizations.py

```diff
@@ -201,6 +204,87 @@ def dot_general_cls(self, mesh_axes: Tuple[str, ...] = ()):
     """Returns dot_general configured with aqt params."""
     return nn.Fp8DotGeneralOp
 
+  def einsum(self, dtype: DType = jnp.float32):
+    return Fp8Einsum(dtype=dtype)
```

Comment:
This looks great! One nit: could you add `einsum` as an abstract method to the `Quantization` class? For consistency? Thanks!

Reply:
Done. PTAL.